src/Select/index.tsx

```diff
@@ -64,21 +107,26 @@ const ListItemWrapper: React.FC<ListItemWrapperProps> = ({
   });
 
   return (
-    <ListItem
-      css={{ alignItems: "baseline" }}
-      key={element.props.value || element.props.children}
-      {...downshiftItemProps}
-      selected={downshiftItemProps["aria-selected"] === "true"}
-      startIcon={
-        selectionIndicator === "checkmark" ? (
-          selected ? (
-            <IconCheck css={{ height: "100%", width: "100%" }} />
-          ) : null
-        ) : undefined
-      }
-    >
-      {element.props.children}
-    </ListItem>
+    <ClassNames>
+      {({ css }) => {
+        return renderListItem(
+          {
+            className: css({ alignItems: "baseline" }),
+            key: element.props.value || element.props.children,
```

Comment:
NBD - is it safe to use value as the key? (Are they guaranteed unique?)

Reply:
Values are _not_ guaranteed to be unique; and I think the resulting bug will produce the correct behavior if you give duplicates values.